build.gradle

```diff
@@ -140,10 +140,14 @@ task downloadAndInstallJRuby(dependsOn: verifyFile, type: Copy) {
 def jrubyBin = "${projectDir}/vendor/jruby/bin/jruby" +
   (System.getProperty("os.name").startsWith("Windows") ? '.bat' : '')
 
+def gemPath = "${projectDir}/vendor/bundle/jruby/2.3.0"
```

Comment:
@colinsurprenant this will leak IT build gems into the vendored dir, I don't think that's correct without adjusting the artifact task accordingly?

Reply:
Not sure I am following?
But essentially this is what we have in the Ruby bootstrap code in `logstash.lib.sh` https://github.com/elastic/logstash/blob/76c24cf697ec576c0efd90e4ade008226cff7338/bin/logstash.lib.sh#L119